graphile.config.mjs

```diff
@@ -54,6 +55,37 @@ const preset = {
   },
   grafast: {
     explain: true,
+    context: (ctx, args) => {
+      // grab any the auth header, if present, from either the http headers or the websocket connection params
+      const authorization = ctx.node?.req?.headers?.authorization || ctx.ws?.connectionParams?.authorization || ''
```

Comment:
`connectionParams` is the raw JSON object, it doesn't have the keys lowercased. You're using `Authorization` as the header, so this is failing to read the header.